### DISCUSSION TITLE
Tag or icon to denote split

### DISCUSSION THREAD

**Discussion by n0svc** [openaudible/openaudible#975] [Ideas]
This kind of falls under the tags idea, but has another (I think better) option. You have icons to show that a book is not downloaded, downloaded(ing), and converted(ing); but it would be nice to have one that show if a book was split. 

I read that you don't like splitting books, but does make it easier to find one's general spot or go back to the beginning of a chapter when interrupted. So thanks for having the option to split files. I saw a request for auto-split. I don't need that. I actually prefer to only do the ones I will use. I do wish I could tell which ones I've already split without looking through the directories.

Like I mentioned in the title, a tag would also work. If you're already working on adding a tag system that you can sort or filter by that would also work great for this. Though for the future, I think an icon would be better.

**Comment by openaudible** [comment]
Good suggestion…. Can work on a tag and or icon. Feel free to propose what the split book icon could look like. 

It does seem like audio players need a way to find your last listened to position when you accidentally skip to the wrong place.

**Comment by n0svc** [comment]
Well, what is the image size (pixels x pixels)?  I've learned in the past that that can be very restricting of what is possible. Otherwise, I'm not sure: a (different) book icon, a folder, or maybe something divided. The current icons look pretty small so there may not be many options. May just have to do another dot, blue maybe. Just thought of a tiny device icon like a box with a smaller box (display) and '+' for controls. That might fit tiny. 

I was also thinking the other day about suggesting adding 'Books Split:' & maybe 'Splitting:' to the 'Library Status' box, but it already looks so full of stuff. Maybe a future, low-priority idea is show/hide options in settings to allow people to hide some things they don't need so they can make the window smaller.

**Comment by openaudible** [comment]
We use 20x20 and 14x14, and display the larger one for people with high DPI monitors. 
I think we can add a dropdown in the search easily... but don't have much room in the Library Status panel.

**Comment by n0svc** [comment]
Well, not really a creative person, but here's an attempt (attached, hopefully). A tiny device icon to show the book is split and ready for devices. 

![device14](https://user-images.githubusercontent.com/71866722/179892202-1ef3022a-74d0-414b-ba6a-515b6d49e893.png)
![device20](https://user-images.githubusercontent.com/71866722/179892205-26f9731b-c1f9-4c5c-a886-56e9bd29a909.png)
 
Also, I forgot about the dropdown menu. That would work. Probably only need to have the 'Split Books' option since the program isn't accessible during splitting.
